### PR TITLE
Add icons and values.schema.json

### PR DESCRIPTION
**Pull request by trispera**
- Add icons for Rahti Helm webpage
- Add values.schema.json for the Form view (Thanks https://github.com/losisin/helm-values-schema-json)

### CODE REVIEW

File: charts/rstudio/Chart.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: v2
 name: rstudio-helm
 description: RStudio Helm Chart for Rahti platform
   Link to the repo https://github.com/CSCfi/helm-charts
-version: 1.0.0
+version: 1.0.1
 sources:
 - https://github.com/CSCfi/helm-charts
+icon:
```
Comment (lvarin) [inline charts/rstudio/Chart.yaml:8]: empty, mistake?

File: charts/rocketchat/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: rocketchat
-version: 6.4.4
+version: 1.1.1
```
Comment (lvarin) [inline charts/rocketchat/Chart.yaml:3]: Version goes down. The highest version is selected in Openshift and it does not have the values.schema.json file on that version.

**Review by lvarin — CHANGES_REQUESTED** [review]
Icon and two versions

File: charts/spark/Chart.yaml [hunk 3]
```diff
@@ -28,4 +28,4 @@ maintainers:
 name: spark
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/spark
-version: 8.1.6
+version: 1.1.1
```
[the remark this answers is not in the document]
Comment (trispera) [reply charts/spark/Chart.yaml:31]: I wanted to harmonize the versions between our different Helm Charts. I can revert

**Review by lvarin — APPROVED** [review]

**Pull request merged**